wiki/Tournaments/MCNC/2021/en.md

```diff
@@ -0,0 +1,130 @@
+---
+tags:
+  - MCNC
+  - MCNC2021
+---
+
+# osu!mania 4K Chinese National Cup 2021
+
+![MCNC 2021 logo](img/logo.png)
+
+The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) was a country-based osu!mania tournament hosted by the \[Crz\]Team. It was the third installment of the osu!mania 4K Chinese National Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2021-03-01/2021-03-31 |
+| Qualifying Contest | 2021-04-05/2021-04-11 |
+| Live drawings | 2021-04-11 (20:00 UTC+8) |
+| Group Stage - Week 1 | 2021-04-12/2021-04-18 |
+| Group Stage - Week 2 | 2021-04-19/2021-04-25 |
+| Round of 16 | 2021-04-26/2021-05-02 |
+| Quarterfinals | 2021-05-03/2021-05-09 |
+| Semifinals - Week 1 | 2021-05-10/2021-05-16 |
+| Semifinals - Week 2 | 2021-05-17/2021-05-23 |
+| Finals - Week 1 | 2021-05-24/2021-05-30 |
+| Finals - Week 2 | 2021-05-31/2021-06-06 |
+
+## Prizes
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | CNY 2,500, unique profile badge, 4 months of osu!supporter tag |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | CNY 1,500, 2 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | CNY 800, 1 months of osu!supporter tag |
+
+## Organisation
+
+The osu!mania 4K Chinese National Cup 2021 was run by various community members.
```

Comment:
`The osu!mania 4K Chinese National Cup 2021 is run by various osu!mania community members.`

Reply:
Done!